test/test_images/logevents/main.go

```diff
@@ -29,7 +28,7 @@ type Heartbeat struct {
 
 func handler(ctx context.Context, data map[string]interface{}) {
 	metadata := cloudevents.FromContext(ctx)
-	log.Printf("[%s] %s %s: %+v", metadata.EventTime.Format(time.RFC3339), metadata.ContentType, metadata.Source, data)
+	log.Printf("[%s]: %+v", metadata.DataContentType(), data)
```

Comment:
I found the latest `event.go` in `knative/pkg` change the function `FromContext` return value type from `*EventContext` to `LoadContext`.  So there is no `EventTime`, `ContentType` and `Source` in `metadata`.  For run this test, I change the log here. I'm not sure if here is right? @evankanderson @mattmoor

Reply:
You should be able to do the following:

```
metadata := cloudevents.FromContext(ctx).AsV02()
log.Printf("[%s] %s %s: %+v", metadata.Time.Format(time.RFC3339), metadata.ContentType, metadata.Source, data)
```

Reply:
Done